### PR TITLE
We can create, update, read, delete, and toggle to show more description

### PR DESCRIPTION
**Pull request by eobaah**

### CODE REVIEW

File: database.js [hunk 1]
```diff
@@ -13,14 +13,15 @@ const getItem = (id) =>
 const addItem = ({ title, author, description, genre, image }) =>
   db.oneOrNone( "INSERT INTO booksbt (title, author, description, genre, image_url) VALUES ($1, $2, $3, $4, $5)", [title, author, description, genre, image] )
 
-const updateItems = (id, book) => {
-    return db.oneOrNone( "UPDATE booksbt SET title=$2, author=$3, description=$4, genre=$5, image_url=$6 WHERE id=$1", [id, book.title, book.author, book.description, book.genre, book.image_url] )
+const updateBook = (id, book) => {
```
Comment (shakalee14) [inline database.js:16]: nice

File: public/javascripts/script.js [hunk 2]
```diff
@@ -1,34 +1,35 @@
...
-        error: function(err){
-          console.log(err)
-        }
-      })
-    }
-  })
 
-  $('.edit-book').on('click', (event) => {
-    const id = $(event.currentTarget).data('id')
+  // $('.delete-book').on('click', (event) => {
```
Comment (shakalee14) [inline public/javascripts/script.js:3]: delete commented code

File: public/javascripts/script.js [hunk 3]
```diff
@@ -1,34 +1,35 @@
...
 
-    $.get('/' + id, (book) => {
-      $('#edit-form-name').val(book.title)
-      $('#edit-form-author').val(book.author)
-      $('#edit-form-description').val(book.description)
-      $('#edit-form-genre').val(book.genre)
-      $('#edit-form-image').val(book.image_url)
-    })
-  })
+  // $('.edit-book').on('click', (event) => {
```
Comment (shakalee14) [inline public/javascripts/script.js:21]: delete commented code

File: public/javascripts/script.js (hunk 2 above)
Comment (shakalee14) [reply public/javascripts/script.js:3]: what's the purpose of this file now?

File: public/stylesheets/style.css [hunk 4]
```diff
@@ -40,6 +40,10 @@ body {
   height:346px;
 }
 
+.description {
```
Comment (shakalee14) [inline public/stylesheets/style.css:43]: lets delete things we arent using

File: routes/index.js [hunk 5]
```diff
@@ -25,15 +25,19 @@ router.post( '/', ( req, res, next ) => {
...
+  removeBooks(id)
+    .then(() => res.redirect('/'))
 })
 
 router.post( '/edit/:id', ( req, res, next ) => {
   const id  = req.params.id
   const book  = req.body
-  updateItems(id, book)
-  .then(() => res.redirect( '/'))
+  console.log("book inside edit route",book)
```
Comment (shakalee14) [inline routes/index.js:35]: remove console.log

File: routes/index.js [hunk 6]
```diff
@@ -25,15 +25,19 @@ router.post( '/', ( req, res, next ) => {
...
+  updateBook(id, book)
+  .then(() => {
+    console.log('id inside route then ', id)
```
Comment (shakalee14) [inline routes/index.js:38]: remove console.log

File: views/index.pug [hunk 7]
```diff
@@ -3,77 +3,84 @@ extends layout
...
+              .collapse(id=`description${book.id}`)
+                p= book.description
+              .bottom-buttons
+                button.btn.btn-default.edit-book(data-toggle='modal' data-target=`#${bookId}`, data-id=book.id)
+                  span.glyphicon.glyphicon-edit(aria-hidden='true' )
+                  form.deleteContainer(method='POST', action='/delete/'+ book.id)
+                    button.btn.btn-danger.delete-book(type="submit")
+                      span.glyphicon.glyphicon-remove(aria-hidden='true')
+
+            #formModal.modal.fade(tabindex='-1', role='dialog', aria-labelledby='formModalLabel')
```
Comment (shakalee14) [inline views/index.pug:31]: i have been told by Jared that using ids is bad, just fyi! i still use them.

**Review by shakalee14 — COMMENTED** [review]
few comments to clean it up!

File: public/javascripts/script.js (hunk 2 above)
Comment (eobaah) [reply public/javascripts/script.js:3]: We may use it to create some front-end sexiness... carousels or something like that lol

**Pull request merged**